test-resources-localstack/test-resources-localstack-core/src/main/java/io/micronaut/testresources/aws/localstack/LocalStackService.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.testresources.aws.localstack;
```

Comment:
Perhaps Localstack shouldn't belong to an "aws" parent package, just in case they go multi-cloud

Reply:
Yeah we can do that!